static/app/views/performance/transactionSummary/transactionThresholdModal.spec.tsx

```diff
@@ -1,38 +1,28 @@
-import {mountWithTheme} from 'sentry-test/enzyme';
-import {act} from 'sentry-test/reactTestingLibrary';
-import {selectByLabel} from 'sentry-test/select-new';
+import selectEvent from 'react-select-event';
 
+import {act, render, screen, userEvent, waitFor} from 'sentry-test/reactTestingLibrary';
```

Comment:
do we need the `act`?

Reply:
Ostensibly yes, for https://github.com/getsentry/sentry/blob/99cddbe358ad7ac5eb63ec202c226597202dfa83/static/app/views/performance/transactionSummary/transactionThresholdModal.spec.tsx#L61

Reply:
The tests pass, yeah. It's a little suspicious, `ProjectsStore.loadInitialData` is wrapped in `act` ~50% of the time in other tests 🤔